docs/data/base/components/button/UnstyledButtonIntroduction.js

```diff
@@ -1,60 +0,0 @@
-import * as React from 'react';
```

Comment:
These files were not removed when the Tailwind CSS + plain CSS demos were added.